src/smspark/bootstrapper.py

```diff
@@ -350,7 +351,18 @@ def set_yarn_spark_resource_config(self) -> None:
             instance_mem_mb = instance_type_info["MemoryInfo"]["SizeInMiB"]
             instance_cores = instance_type_info["VCpuInfo"]["DefaultVCpus"]
             logging.info(
-                f"Detected instance type: {instance_type} with "
+                f"Detected instance type for processing: {instance_type} with "
+                f"total memory: {instance_mem_mb}M and total cores: {instance_cores}"
+            )
+        elif is_training_job() and instance_type_info:
```

Comment:
Why a training specific branch, whats the delta that is not available in Processing right now? May be we can make that available via Processing in the algo container.